### PR TITLE
Save upload by name

### PR DESCRIPTION
**Pull request by olenagerasimova**
Made `Multipart` and `WheelSlice` save uploads by the correct keys into storage.

### CODE REVIEW

File: src/main/java/com/artipie/pypi/http/Multipart.java [hunk 1]
```diff
@@ -50,6 +54,11 @@
      */
     private static final int BUFFER = 4096;
 
+    /**
+     * Uploaded file name.
```
Comment (g4s8) [inline src/main/java/com/artipie/pypi/http/Multipart.java:58]: @olenagerasimova it's not clear what is this filename. I'd say that it's some header value with filename.

File: src/main/java/com/artipie/pypi/http/WheelSlice.java [hunk 2]
```diff
@@ -43,6 +45,11 @@
  */
 final class WheelSlice implements Slice {
 
+    /**
+     * Package name.
+     */
+    private static final Pattern PKG_NAME = Pattern.compile("(.*?)-.*");
```
Comment (g4s8) [inline src/main/java/com/artipie/pypi/http/WheelSlice.java:51]: @olenagerasimova same here: it's not clear what is this PKG_NAME

**Comment by olenagerasimova** [comment]
@g4s8 thanks, extended javadoc

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.3`

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.3`

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.3`

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.3`

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.3`

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.3`

**Pull request merged**